src/utils/map.ts

```diff
@@ -60,21 +60,25 @@ export module Utils {
     }
 
     /**
-     * Return an array of the values in the Map
+     * The forEach method executes the provided callback once for each key of the map which
+     * actually exist. It is not invoked for keys which have been deleted.
+     * However, it is executed for values which are present but have the value undefined.
      *
-     * @return {V[]} The values in the store
-     */
-    public values() {
-      return this._keyValuePairs.map((keyValuePair) => keyValuePair.value);
-    }
-
-    /**
-     * Return an array of keys in the Map.
+     * Callback is invoked with three arguments:
+     *   - the element value
+     *   - the element key
+     *   - the Map object being traversed
```

Comment:
Oh I mean, let's add a small snippet and then just link to MDN for the full description